templates/myaccount/subscription-totals-table.php

```diff
@@ -77,7 +76,8 @@
 				<?php
 			}
 
-			if ( $subscription->has_status( array( 'completed', 'processing' ) ) && ( $purchase_note = get_post_meta( $_product->id, '_purchase_note', true ) ) ) {
+			$purchase_note = get_post_meta( $_product->get_id(), '_purchase_note', true );
```

Comment:
Just flagging that this could be updated to `$_product->get_purchase_note()` but there's a lot of post meta calls related to products so I think this is fine to leave it for now given this is how the code was prior to your changes and you're just fixing a coding standards violation.